packages/router-generator/src/generator.ts

```diff
@@ -462,6 +462,7 @@ export async function generator(config: Config) {
     '// @ts-nocheck',
     '// noinspection JSUnusedGlobalSymbols',
     '// This file is auto-generated by TanStack Router',
```

Comment:
I would put all this into the default value of `routeTreeFileHeader`

Reply:
Will do, although I think its worth keeping this one outside the default list and something we ship with the generator always instead: // This file is auto-generated by TanStack Router